### DISCUSSION TITLE
How to Discern HLG or PQ Rendition?

### DISCUSSION THREAD

**Discussion by blue-j** [colour-science/colour#1260] [Q&A]
I have a JXL file that is 16bpc, in Rec.2020, and is using an [Adobe gain map](https://helpx.adobe.com/camera-raw/using/gain-map.html).  I can discern that the file is in 2020 by looking at the primaries and white point, but how to discern which EOTF it is using?  Adobe demo gain map app reports that it uses PQ.

- J

[08.jxl.zip](https://github.com/colour-science/colour/files/15171371/08.jxl.zip)

**Comment by blue-j** [comment]
i was able to do this using the _jxlinfo_ utility provided by the library.  so the pressure is off for me!  but... if anyone has a suggestion on how we could determine this, that would be very informative!  - J

**Reply by thomasmansencal** [reply]
PQ stands for Perceptual Quantizer. It is ST.2084.
